libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -0,0 +1,231 @@
+<!--
```

Comment:
Consider splitting this into multiple components, since the template is quite large. Ex. a component handling the form

Reply:
I think this can wait..